src/doc/rustc/src/command-line-arguments.md

```diff
@@ -10,6 +10,11 @@ This flag will print out help information for `rustc`.
 
 This flag can turn on or off various `#[cfg]` settings.
 
+The value can either be a single identifier or two identifiers separated by `=`.
+
+For examples, `--cfg 'verbose'` or `--cfg 'feature=serde'`. These correspond
```

Comment:
`--cfg feature=serde` is incorrect, you need a string literal for the value - `--cfg feature = "serde"`.

Reply:
`--cfg` argument (after shell preprocessing, etc) is literally a meta-item, same thing as in `#[cfg(feature = "serde")]` and with the same restrictions.

Reply:
Whoops. Thought I put those quotes in there. Should have just copy-pasted from the `cargo build --verbose` output...